Add privacy and UX related options

rkfg opened the pull request:
Allow to disable showing read receipts and showing/sending type notifications, disable markdown with a visual setting (currently is only possible via undocumented /markdown off command), change the message font size.

ylecollen commented:
i'm implementing the languages selection dialog after i will merge your PR (i will need to modify few things)

rkfg commented:
I've fixed the preferences keys being locale-dependent and added a couple of new settings for media. Now it only requires 3 taps (if you set the preferred defaults) to take and post a photo instead of 5. Much more convenient.

rkfg commented:
Oh, and since the contributing guide requires sign off and allows to put it to the pull request comment I'll drop a line here:
`Signed-off-by: Sergey Shpikin <rkfg@rkfg.me>`

rkfg commented:
Removed obsolete settings and regrouped the rest. Seems that they need to be ported to PreferencesManager.

bmarty commented:
Hi @rkfg ,

Seem that your PR has been forgotten, sorry about that. Riot does contains a font size settings now, and the code has changed a lot since you've submitted the PR.

I'm afraid it will be hard to rebase it.

What do you want to do with this PR?

Benoît

rkfg commented:
Thanks for your attention. This patch is mostly obsolete, however I'd still like to have some bits of it implemented: the default media source and default compression settings. It would make posting photos so much more convenient. For now I always have to select the source (photo), take a picture, then select the same compression (medium). Having a preference for those would be nice and it's really easy to implement like I already did once. Other than that this PR can be closed safely.
